test/TestCase.php

```diff
@@ -15,45 +15,75 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2018-2021 (original work) Open Assessment Technologies SA;
+ * Copyright (c) 2018-2022 (original work) Open Assessment Technologies SA.
  */
 
+declare(strict_types=1);
+
 namespace oat\generis\test;
 
-use oat\oatbox\service\ServiceManager;
 use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
 use Psr\Container\ContainerInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use oat\oatbox\service\ServiceManager;
 use PHPUnit\Framework\TestCase as UnitTestCase;
 
+/**
+ * @deprecated Use \PHPUnit\Framework\TestCase instead
+ */
 abstract class TestCase extends UnitTestCase
 {
     use SqlMockTrait;
 
     /**
-     * @param array $services
-     * @return ServiceLocatorInterface|ServiceManager
+     * @deprecated Use \oat\generis\test\ServiceManagerMockTrait::getServiceManagerMock() instead
```

Comment:
We may add extra info in the deprecation explaining we do not want to use Prophecy anymore to reduce the number of dependencies since PhpUnit does all the job. Wdyt?